tools/arch-visualizer/viewer/src/components/DetailPanel.tsx

```
@@ -0,0 +1,947 @@
+              )}
+              {component.port && (
+                <span className={`text-xs font-mono ${darkMode ? "text-blue-400" : "text-blue-600"}`}>
+                  :{component.port}
+                </span>
+              )}
+            </div>
+          </div>
+          <button
+            onClick={closeDetail}
+            className={`p-1 rounded-lg transition-colors ${darkMode ? "hover:bg-zinc-800 text-zinc-500" : "hover:bg-zinc-100 text-zinc-400"}`}
+          >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)